<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HF Fine-Tune</string>
  </property>
  <widget class="QPushButton" name="pushButton_weav_refresh">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_weav_status">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_host">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>http://localhost:8080</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_host">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_class">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>145</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_count">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Count records</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_key">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_key">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>65</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>600</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_filter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>205</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;{&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &amp;quot;path&amp;quot;: &amp;quot;user_id&amp;quot;,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &amp;quot;operator&amp;quot;: &amp;quot;Equal&amp;quot;,&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  &amp;quot;valueText&amp;quot;: &amp;quot;108249722086702971940&amp;quot;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_filter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_schema">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Schema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Custom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_process">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_schema_add">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Property</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_populate_thread_id">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Populate thread_id</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>%p% [%v / %m]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_count_distinct">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Count distinct</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_field">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>175</y>
     <width>421</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>thread_id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Field</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mongo_create_threads_collection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Mongo cr threads</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_schema_add_class">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Class</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
